src/components/safe-apps/AppFrame/useAppCommunicator.ts

```diff
@@ -103,13 +103,8 @@ const useAppCommunicator = (
             },
           )
         },
-        onError: (error, data) => {
-          logError(Errors._901, error.message, {
-            contexts: {
-              safeApp: app || {},
-              request: data,
-            },
```

Comment:
This made absolutely no difference, as logError doesn't send anything to Sentry. Only trackError does.